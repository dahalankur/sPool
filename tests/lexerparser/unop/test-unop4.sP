# arith neg with parens
-(23 + 45)
-(45 * (7 - 3 + -(3 / 4)))
