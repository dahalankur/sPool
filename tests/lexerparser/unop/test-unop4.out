PROGRAM {[ 
    UNOP(NEG, BINOP(INT(23), PLUS, INT(45))) 
    UNOP(NEG, BINOP(INT(45), TIMES, BINOP(BINOP(INT(7), MINUS, INT(3)), PLUS, UNOP(NEG, BINOP(INT(3), DIV, INT(4))))))]}
